### PR TITLE
Feature/SPOT-2: User can see sign up & sign in UI

### PR DESCRIPTION
**Pull request by dattruong2110**

### CODE REVIEW

**Comment by HuynhNguyen1605** [comment]
Good

File: src/components/login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  };
+
+  const handleSignInWithGoogle = () => {
+    signInWithPopup(auth, provider).then((data) => {
+      setValue(data.user.email);
+      localStorage.setItem("email", data.user.email);
+    });
+  };
+
+  // useEffect(() => {
```
Comment (daonguyendev) [inline src/components/login/Login.jsx:54]: Should remove redundant code

File: src/components/login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleValidate = () => {
+    const errors = {};
+    if (!form.email) {
+      errors.email = "Required";
+    } else if (!REGEX.email.test(form.email)) {
+      errors.email = "Invalid email address";
```
Comment (daonguyendev) [inline src/components/login/Login.jsx:28]: Should create constant for messages

File: src/components/login/Login.jsx (hunk 1 above)
Comment (dattruong2110) [reply src/components/login/Login.jsx]: Resolved

File: src/components/login/Login.jsx (hunk 2 above)
Comment (dattruong2110) [reply src/components/login/Login.jsx]: Resolved

**Review by daonguyendev — APPROVED** [review]
LGTM

**Review by HuynhNguyen1605 — APPROVED** [review]
Good

**Pull request merged**